upgrade_analysis/odoo_patch/odoo_patch.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class OdooPatch(object):
```

Comment:
I also think this code is a good candidate to exist inside `openupgradelib` instead.

Reply:
In the current simplistic design, it is not separable from the set of patches that it serves.